source/jsonnet/england-wales/census_individual.jsonnet

```diff
@@ -92,6 +92,8 @@ local mainly_work_outside_uk = import 'individual/blocks/employment/mainly_work_
 local supervise = import 'individual/blocks/employment/supervise.jsonnet';
 local work_travel = import 'individual/blocks/employment/work_travel.jsonnet';
 
+local routingRuleSummaryElement = 'group';
+local routingRuleSummaryValue = 'submit-group';
```

Comment:
I don't understand why this can't be `section` and `end`, as in the absence of the hub that should then route to the first block of the next section, which would be the final summary.

Reply:
I hadn't thought of it like that, I will see if that works, would remove a shed load of logic if it did